homework-classes/ContributorsView.js

```diff
@@ -19,8 +19,46 @@
      * @param {Object[]} contributors An array of contributor objects
      */
     render(contributors) {
-      // TODO: replace this comment and the console.log with your own code
       console.log('ContributorsView', contributors);
+      const contributorsContainer = document.querySelector(
+        '.contributors-container',
+      );
+      const select = document.querySelector('.repo-select');
+      const p = createAndAppend('p', contributorsContainer, {
+        text: 'Contributions',
+      });
+
+      async function createContributorsSection(contributors, parent) {
```

Comment:
This need not be an `async` function, You are not using `await` inside the function body. Fetching data is the responsibility of the **Model** class. Therefore, the `try/catch` block is also unwanted. In fact, the responsibility to render error is that of the `ErrorView` class.

Finally, this function should not be a function at all as you call immediately after its definition (see line 57).